src/main/java/com/uid2/admin/vertx/AdminVerticle.java

```diff
@@ -147,6 +161,11 @@ private void handleTokenGet(RoutingContext rc) {
         });
     }
 
+    private void respondWithTestAdminUser(RoutingContext rc) {
+        // This test user is set up in src/main/resources/localstack/s3/admins/admins.json
+        handleEmailContactInfo("test.user@thetradedesk.com", rc);

```

Comment:
Please change this to `test.user@uidapi.com`.

Reply:
addressed